front/components/assistant/conversation/Conversation.tsx

```diff
@@ -209,6 +216,8 @@ export default function Conversation({
                 </div>
               </div>
             );
+          case "content_fragment":
+            return null;
```

Comment:
@spolu made the assumption that we don;t want to display for now

Reply:
Yep